diff --git a/open_instruct/eval/ace/__pycache__/run_eval.cpython-39.pyc b/open_instruct/eval/ace/__pycache__/run_eval.cpython-39.pyc
index 21ee6b8..271bcff 100644
Binary files a/open_instruct/eval/ace/__pycache__/run_eval.cpython-39.pyc and b/open_instruct/eval/ace/__pycache__/run_eval.cpython-39.pyc differ
diff --git a/open_instruct/open_instruct/__pycache__/llama_flash_attn_monkey_patch.cpython-38.pyc b/open_instruct/open_instruct/__pycache__/llama_flash_attn_monkey_patch.cpython-38.pyc
index 003357f..91a8a34 100644
Binary files a/open_instruct/open_instruct/__pycache__/llama_flash_attn_monkey_patch.cpython-38.pyc and b/open_instruct/open_instruct/__pycache__/llama_flash_attn_monkey_patch.cpython-38.pyc differ
diff --git a/scripts/PKU_finetune_GenData_Val.sh b/scripts/PKU_finetune_GenData_Val.sh
index e1c6654..8f8933b 100644
--- a/scripts/PKU_finetune_GenData_Val.sh
+++ b/scripts/PKU_finetune_GenData_Val.sh
@@ -1,4 +1,4 @@
-export CUDA_VISIBLE_DEVICES=4,5
+export CUDA_VISIBLE_DEVICES=4,5,6,7
 
 MODEL_PATH="/home/models/Llama-2-7b-hf/"
 OUTPUT_NAME=Llama2_GenData_1definitions_v2/
@@ -9,9 +9,9 @@ REPORT_TAGS="CtrlGen"
 
 # ceildiv(){ echo $((($1+$2-1)/$2)); }
 # NUM_GPUS=$(ceildiv ${#CUDA_VISIBLE_DEVICES} 2)
-NUM_GPUS=2
+NUM_GPUS=4
 BATCH_SIZE_PER_GPU=1
-TOTAL_BATCH_SIZE=2
+TOTAL_BATCH_SIZE=4
 GRADIENT_ACC_STEPS=$(($TOTAL_BATCH_SIZE/$NUM_GPUS/$BATCH_SIZE_PER_GPU))
 echo "Training model ${MODEL_PATH} using $NUM_GPUS GPUs, $BATCH_SIZE_PER_GPU batch size per GPU, $GRADIENT_ACC_STEPS gradient accumulation steps"
 
diff --git a/scripts/pku_generated_training_data_1definitions.py b/scripts/pku_generated_training_data_1definitions.py
index 6228412..fe4f0f7 100644
--- a/scripts/pku_generated_training_data_1definitions.py
+++ b/scripts/pku_generated_training_data_1definitions.py
@@ -110,7 +110,7 @@ for parent_event in train_parent_list:
                             # "parent": parent_event,
                             # "events": events,
                             # "sons": sons,
-                            "prompt": f"SENTENCE: {sentence} \n EVENT TYPE: {event_name}. \n DEFINITION: {event_definition} \n PARENT: {parent_event}, SON: {text_sons}. \n So what is the trigger?",
+                            "prompt": f"SENTENCE: {negative_sample} \n EVENT TYPE: {event_name}. \n DEFINITION: {event_definition} \n PARENT: {parent_event}, SON: {text_sons}. \n So what is the trigger?",
                             "completion": f"Event trigger is <trigger>."
                             })
         else:
@@ -219,7 +219,7 @@ for parent_event in val_parent_list:
                             # "parent": parent_event,
                             # "events": events,
                             # "sons": sons,
-                            "prompt": f"SENTENCE: {sentence} \n EVENT TYPE: {event_name}. \n DEFINITION: {event_definition} \n PARENT: {parent_event}, SON: {text_sons}. \n So what is the trigger?",
+                            "prompt": f"SENTENCE: {negative_sample} \n EVENT TYPE: {event_name}. \n DEFINITION: {event_definition} \n PARENT: {parent_event}, SON: {text_sons}. \n So what is the trigger?",
                             "completion": f"Event trigger is ",
                             "trigger":  "<trigger>"
                             })
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index da0a5ca..3d9a599 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20231105_231424-6mtowpzv/logs/debug-internal.log
\ No newline at end of file
+run-20231106_024104-mus9k2ax/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index e9e0865..1a8560c 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20231105_231424-6mtowpzv/logs/debug.log
\ No newline at end of file
+run-20231106_024104-mus9k2ax/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index f7667d3..ded47f9 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20231105_231424-6mtowpzv
\ No newline at end of file
+run-20231106_024104-mus9k2ax
\ No newline at end of file
